bin/strap.sh

```diff
@@ -21,7 +21,26 @@ if [ "$1" = "--sudo-wait" ]; then
   exit 0
 fi
 
-[ "$1" = "--debug" ] && STRAP_DEBUG="1"
+#arg parsing
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -d|--debug)
+      STRAP_DEBUG="1"
+      shift
+      ;;
+    -k|--parallels-key)
+      # write license key to file
+      echo -n "$2" > "$HOME/.parallels-lk"
```

Comment:
Is there a check you can do here to make sure $2 exists?  Maybe even a simple regex on the value itself if possible (I'd have to look up my key).

Reply:
yes, i'll add that